www/%team/distributing/index.html.spt

```diff
@@ -1,10 +1,19 @@
 # encoding: utf8
+
+from aspen import Response
 from gratipay.utils import get_team
 
 [-----------------------------------------------------------------------------]
 team = get_team(state)
 if team.available == 0:
     website.redirect('..', base_url='')
+
+if user.participant == None:
+    raise Response(401)
+
+if not (user.ADMIN or user.participant.member_of(team) or user.participant.username == team.owner):
+    raise Response(403)
+
```

Comment:
Sorry (and I tried to get ahead of this at https://github.com/gratipay/gratipay.com/pull/4077#discussion_r73424777), but I don't think we should restrict the distributing page. Why _would_ we?